packages/patch/patch.1.0.0/opam

```diff
@@ -12,6 +12,7 @@ depends: [
   "dune"
   "alcotest" {with-test}
   "crowbar" {with-test}
+  "base-bytes"
```

Comment:
The issue with this is that the package now suddenly depends on ocamlfind which might not be the intent of the developers. `patch.1.0.1` works fine with ocaml5, why not simply add the `< "5.0"` constraint?

Reply:
I asked what was the right fix for that and @dra27 told me to use `base-bytes`. But as you prefer, I don't think it matters in that case.

Reply:
The current consensus is to go half way:
```suggestion
  "base-bytes" {ocaml:version >= "5.0"}
```